WORKSPACE

```diff
@@ -46,28 +93,17 @@ local_repository(
     path = "hw/ip/prim/util/vendor/google_verible_verilog_syntax_py",
 )
 
+################################################################################
+# Third Party SW
+################################################################################
 # Abseil is required by googletest.
 http_archive(
     name = "com_google_absl",
     strip_prefix = "abseil-cpp-master",
     urls = ["https://github.com/abseil/abseil-cpp/archive/master.zip"],
 )
 
-# Buildifier is a linting tool for our WORKSPACE and BUILD files
-http_archive(
-    name = "io_bazel_rules_go",
-    sha256 = "d1ffd055969c8f8d431e2d439813e42326961d0942bdf734d2c95dc30c369566",
-    urls = [
-        "https://github.com/bazelbuild/rules_go/releases/download/v0.24.5/rules_go-v0.24.5.tar.gz",
-    ],
-)
-
-load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
-
-go_rules_dependencies()
-
-go_register_toolchains()
-
+# Bazel Gazelle
```

Comment:
I don't think this header is highlighting a topic and the name of the archive is already helpful to tell us what this is. The old structure indicated why it was here (it got added to support buildifier) but I don't think that needs to be recorded in the file itself.